src/Mvc/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.props

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project>
+  <!--
+    Settings users may update as they see fit.
+  -->
+  <PropertyGroup>
+    <!--
+      Options added to the Open API retrieval tool ('dotnet-getdocument') command line. Available options control
+      console output: 'no-color', 'prefix-output' and 'verbose'. All require a double-dash prefix.
+    -->
+    <OpenApiGetDocumentOptions Condition=" '$(OpenApiGetDocumentOptions)' == '' "></OpenApiGetDocumentOptions>
```

Comment:
This line doesn't do anything. "If OpenApiGetDocumentOptions is empty, assign OpenApiGetDocumentOptions  to empty."

Reply:
Good catch! This property should be `true` by default

Reply:
Oh, wait. This is here only for documentation purposes. Should be empty by default.